### DISCUSSION TITLE
Error 259 with powershell catbox

### DISCUSSION THREAD

**Discussion by franco-cardoso** [s0hv/foo_discord_rich#24] [Q&A] [answered]
`Discord Rich Presence Integration v1.3.0-beta-6+4aec8f6: artwork uploader exited with status: 259 and error: Process timed out`

The powershell catbox uploader script is always timing out, this isn't an upload speed issue because i tried with a 20kb image and the result was the same. Execution policy is unrestricted and i made sure to input the proper command using powershell 7

**Comment by s0hv** [comment]
Could you try manually running the script for a cover art image. It can be done with the following command
```
echo "C:\path\to\image" | "C:\Program Files\PowerShell\7\pwsh.exe" "C:\path\to\catbox.ps1"
```
and replacing the image and catbox script locations. If it takes long you need to increase the upload script timeout which can be found under `Preferences -> Tools -> Discord Rich Presence Integration -> Advanced`

**Comment by franco-cardoso** [comment]
i get this:
> Resolve-Path: C:\catbox.ps1:4
Line |
   4 |  $upload = Resolve-Path -LiteralPath $input | Select-Object -ExpandPro …
     |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find drive. A drive with the name '"C' does not exist.
Get-Item: C:\catbox.ps1:8
Line |
   8 |      "fileToUpload"=Get-Item $upload
     |                              ~~~~~~~
     | Cannot bind argument to parameter 'Path' because it is null.
Invoke-WebRequest: C:\catbox.ps1:10
Line |
  10 |  Invoke-WebRequest -Uri $uri -Method POST -Form $body | Select-Object  …
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | No request type given?

**Reply by s0hv** [reply]
Looks like there's a problem with with the image path. Make sure you don't have any invisible characters before C:

**Reply by franco-cardoso** [reply]
i moved the image to a simpler path just in case and the result was the same, copy pasting the path into my file explorer does open the image normally

**Reply by s0hv** [reply]
How about modifying the script and inserting the path there manually

**Reply by franco-cardoso** [reply]
that does work

**Reply by s0hv** [reply]
You could try increasing the timeout from the settings then if the script itself works

**Reply by franco-cardoso** [reply, marked as answer]
i increased it to a couple minutes and it still timed out. i tried switching to the python script instead though and that one does work. regardless, thanks a lot for your help!
